.ibm/pipelines/jobs/operator.sh

```diff
@@ -10,7 +10,14 @@ install_rhdh_operator() {
   rm -f /tmp/install-rhdh-catalog-source.sh
   curl -L https://raw.githubusercontent.com/redhat-developer/rhdh-operator/refs/heads/main/.rhdh/scripts/install-rhdh-catalog-source.sh > /tmp/install-rhdh-catalog-source.sh
```

Comment:
Wouldn't it make more sense to also download the script from the appropriate release branch?

Reply:
Yeah, that's true, I'm changing it to use the correct branch. Thanks